### ISSUE TITLE
KL loss issue

### ISSUE THREAD

**Issue by sharkDDD** [yifan123/flow_grpo#111]
`kl_loss = ((prev_sample_mean - prev_sample_mean_ref) ** 2).mean(dim=(1,2,3), keepdim=True) / (2 * std_dev_t ** 2)`

Thank you for your work. I encountered an error when running the KL loss computation in the code. The shapes of my prev_sample_mean and prev_sample_mean_ref are both [batch_size, 1024, 64]. When trying to calculate the mean across dim=1, 2, and 3, an error occurs. I’ve checked the preceding operations and didn’t find any issues. May I ask if I’ve overlooked any important steps, or could there be a problem with the data processing?
